py-polars/polars/expr/datetime.py

```diff
@@ -885,6 +933,33 @@ def datetime(self) -> Expr:
         -------
         Expr
             Expression of data type :class:`Datetime`.
+
+        Examples
+        --------
+        >>> from datetime import datetime
+        >>> from pytz import utc
```

Comment:
We don't use `pytz`, so let's not rely on it for docs.

You can define `schema={"datetime UTC": pl.Datetime(time_zone="UTC")}` to make the data timezone aware.

@MarcoGorelli is this indeed the point of this method - to strip the time zone? If so, we should probably give it a more suitable name 🤔

Reply:
I removed `pytz` from the example. 

I don't know for certain what the intended purpose of the method is, but I was looking at the [docs for the analogous series method](https://docs.pola.rs/py-polars/html/reference/series/api/polars.Series.dt.datetime.html#polars.Series.dt.datetime) for reference, which also implies that the purpose is to strip the time zone.

Reply:
I'll merge this - we can update the example / method name based on Marco's input later if required.

Reply:
yeah this just returns the local datetime

there was some discussion here https://github.com/pola-rs/polars/issues/6130

(happy to revisit if you like though!)

Reply:
So currently it's the same as `.dt.convert_time_zone(None)`, right? That is much more explicit than `.dt.datetime()` which apparently drops the time zone (I would have to look that up in the docs if I didn't know it).

I think it can probably be removed, but if you see a good case for keeping it then I don't mind keeping it around.

Reply:
it would be `.dt.replace_time_zone(None)`

so, remove `.dt.datetime` but keep `.dt.date` and `.dt.time`? yeah no objections